reco_utils/dataset/url_utils.py

```diff
@@ -7,6 +7,25 @@
 
 log = logging.getLogger(__name__)
 
+from tqdm import tqdm
+
+
+class TqdmUpTo(tqdm):
```

Comment:
do we make use of the update_to functionality?

Reply:
yes, this was used in maybe_download